### PR TITLE
Add Hindi translation

### PR DESCRIPTION
**Pull request by tupaschoal**

### CODE REVIEW

**Review by tupaschoal — COMMENTED** [review]
@shivam-maurya-git there are a few sentences that are still in English, are you still working on the translation?

**Comment by shivam-maurya-git** [comment]
@tupaschoal , Not sentences; 7 words are in English; they work better if they are in English. And, translation in Hindi is completed.

File: _data/trans/hi.json [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+    "defaultnote_easy": "दिए हुए पेज पर जाए और अपना अकाउंट मिटाए |",
+    "defaultnote_email": "दिए गए पते पर अकाउंट मिटाने के लिए ईमेल करे |",
+    "difficulty": "स्तर",
+    "difficulty_easy": "आसान",
+    "difficulty_hard": "कठिन",
+    "difficulty_impossible": "असंभव",
+    "difficulty_limited": "बहुत मुश्किल",
+    "difficulty_medium": "मध्यम",
+    "extension_browser": "क्रोम या फायर फॉक्स एक्सटेंशन",
+    "extensionguide": "Extesnion के बिंदुओं का मतलब",
```
Comment (tupaschoal) [inline _data/trans/hi.json:13]: @shivam-maurya-git should this be "Extension", like below?

File: _data/trans/hi.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "guideeasy": "आसान प्रक्रिया",
+    "guideexplanations": "ऊपर दिए गए लिंक पर एक रंग बिंदु है, जो अकाउंट मिटाने करने के प्रक्रिया की कठनाई का स्तर बताता है |",
+    "guidehard": "पूरी तरह से अकाउंट मिटने के लिए ग्राहक सेवा केन्द्र से संपर्क करना पड़ेगा",
+    "guideimpossible": "मिटाया नहीं जा सकता है",
+    "guidelimited": "अगर आपके क्षेत्र मतलब देश या राज्य मे निजता के अधिकार लागू होंगे, तभी आप अकाउंट मिटा सकते है",
+    "guidemedium": "कुछ और प्रक्रियाएं पूरी करनी होंगी",
+    "hideinfo": "जानकारी छुपाए...",
+    "jgmd": "अपना डाटा पाने के लिए जाए",
+    "name": "English",
+    "noinfo": "No Info Available",
```
Comment (tupaschoal) [inline _data/trans/hi.json:30]: @shivam-maurya-git is this best not translated?

File: _data/trans/hi.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    "guide": "दिशानिर्देश",
+    "guideeasy": "आसान प्रक्रिया",
+    "guideexplanations": "ऊपर दिए गए लिंक पर एक रंग बिंदु है, जो अकाउंट मिटाने करने के प्रक्रिया की कठनाई का स्तर बताता है |",
+    "guidehard": "पूरी तरह से अकाउंट मिटने के लिए ग्राहक सेवा केन्द्र से संपर्क करना पड़ेगा",
+    "guideimpossible": "मिटाया नहीं जा सकता है",
+    "guidelimited": "अगर आपके क्षेत्र मतलब देश या राज्य मे निजता के अधिकार लागू होंगे, तभी आप अकाउंट मिटा सकते है",
+    "guidemedium": "कुछ और प्रक्रियाएं पूरी करनी होंगी",
+    "hideinfo": "जानकारी छुपाए...",
+    "jgmd": "अपना डाटा पाने के लिए जाए",
+    "name": "English",
```
Comment (tupaschoal) [inline _data/trans/hi.json:29]: @shivam-maurya-git this should be the language name as it reads for native speakers

File: _data/trans/hi.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    "reset": "Reset",
+    "search": "Search",
+    "sendmail": "ईमेल भेजे",
+    "showinfo": "जानकारी दिखाए",
+    "tagline": "इंटरनेट की के अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह",
+    "title": "Just Delete Me | इंटरनेट के कई अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह",
+    "twitter": "JDM के बारे मे ट्वीट करे",
+    "whatisthis": "यह क्या है?",
+    "whatisthis1": "Many companies use",
+    "whatisthis2": "dark pattern",
```
Comment (tupaschoal) [inline _data/trans/hi.json:44]: This is the beginning of the phrase 

>"Many companies use [dark pattern](https://darkpatterns.org/) techniques to make it difficult to find how to delete your account. JustDeleteMe aims to be a directory of urls to enable you to easily delete your account from web services."

@shivam-maurya-git sure it should not be translated?

File: _data/trans/hi.json (hunk 1 above)
Comment (shivam-maurya-git) [reply _data/trans/hi.json:13]: Yes, it is better. Hindi Speakers understand this much English ( Technical Words). Hindi translation of  "extension" will be hard word for readers.

File: _data/trans/hi.json (hunk 3 above)
Comment (shivam-maurya-git) [reply _data/trans/hi.json:29]: Yeah, I gone through website and I dubbed this words according to role of these words on JDM website. Direct translation will not be much effective.

File: _data/trans/hi.json (hunk 2 above)
Comment (shivam-maurya-git) [reply _data/trans/hi.json:30]: I forgotten to save translation, now I saved it. Now, it is updated.

File: _data/trans/hi.json (hunk 4 above)
Comment (shivam-maurya-git) [reply _data/trans/hi.json:44]: I forgotten to save translation, now I saved it. Now, it is updated.

File: _data/trans/hi.json [hunk 5]
```diff
@@ -26,22 +26,22 @@
...
+    "search": "खोजें",
     "sendmail": "ईमेल भेजे",
     "showinfo": "जानकारी दिखाए",
     "tagline": "इंटरनेट की के अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह",
     "title": "Just Delete Me | इंटरनेट के कई अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह",
     "twitter": "JDM के बारे मे ट्वीट करे",
     "whatisthis": "यह क्या है?",
-    "whatisthis1": "Many companies use",
-    "whatisthis2": "dark pattern",
-    "whatisthis3": "प्रयोग करती है ताकि अकाउंट मिटाना मुश्किल हो जाए | JustDeleteMe का लक्ष्य इंटरनेट के कई सारे अकाउंट डिलीट करने के लिए सीधे तरीके का संग्रह बनाना है |",
```
Comment (shivam-maurya-git) [inline _data/trans/hi.json]: Restore whatisthis3 and everything else is correct.

**Review by shivam-maurya-git — CHANGES_REQUESTED** [review]

**Review by shivam-maurya-git — APPROVED** [review]

**Comment by tupaschoal** [comment]
It is merged now, thanks for the work @shivam-maurya-git !

**Review by Hellothuy1 — APPROVED** [review]

**Pull request merged**